whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/MntLowerAttributeValidationSpec.groovy

```diff
@@ -390,12 +390,13 @@ class MntLowerAttributeValidationSpec extends BaseQueryUpdateSpec {
         )
 
         then:
-        ack.success
+        ack.failed
```

Comment:
modify inetnum (assigned pi and co-maintained): **add mnt-lower** of lir mnt by lir